addon-test-support/@ember/test-helpers/dom/-target.ts

```diff
@@ -20,3 +27,18 @@ export function isDocument(target: any): target is Document {
 export function isContentEditable(element: Element): element is HTMLElementContentEditable {
   return 'isContentEditable' in element && (element as HTMLElement).isContentEditable;
 }
+
+/**
+  Used internally by the DOM interaction helpers to find either window or an element.
+
+  @private
+  @param {string|Element} target the window, an element or selector to retrieve
+  @returns {Element|Window} the target or selector
+*/
+export function getTarget(target: Target): Element | Document | Window | null {
```

Comment:
Hmm, `getTarget` seems odd to me (as a name here). Maybe `getWindowOrElement` (definitely makes more sense to what it is)?

Reply:
Agreed.

Reply:
Renamed.